examples/hello_world/run.cc

```diff
@@ -37,7 +37,7 @@ std::vector<int> tokenize(
   return tokens;
 }
 
-int main(int argc, char** argv) {
+int main(int argc, const char** argv) {
```

Comment:
Why do we use const here? char** or char*[] is standard, see 3.6.1 in https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf.

Reply:
argv from command  params just read, so want use const char**,   want use libgemma util/app.h this function `gcpp::LoaderArgs loader(argc, argv)`  i want to cast const char** to  char** ,    like this `char** nonConstPtr = const_cast<char**>(constPtr);`;  but this command argv is immutable

Reply:
> Why do we use const here? char** or char*[] is standard, see 3.6.1 in https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2012/n3337.pdf.

use `main(int argc, char** argv)` it's ok~